docs/src/main/sphinx/sql/alter-table.md

```diff
@@ -54,13 +54,14 @@ per-connector basis.
 You can use the `=>` operator for passing named parameter values.
 The left side is the name of the parameter, the right side is the value being passed.
 
-Collapse files in a table that are over 128 megabytes in size with the `optimize`
-procedure from the [Hive](hive-alter-table-execute), [Delta
-Lake](delta-lake-alter-table-execute) , and
-[Iceberg](iceberg-alter-table-execute) connectors:
+Executable commands are contributed by connectors, for example, to merge files
+in a table called `test_table` that is over 128 megabytes in size within the
```

Comment:
This has to be reworded ..

its not the table that is over 128mb .. its about merging all the many data files in that table that are smaller than 128mb into fewer files that are larger than 128mb

So the initial suggestion from @findinpath is probably better .. but also a bit terse and assuming a lot of knowledge. 

Lastly .. do you think 128mb file size is a reasonable example @findinpath ?

Reply:
> Lastly .. do you think 128mb file size is a reasonable example @findinpath ?

I'm missing field experience here.

@raunaqmorarka can you help on this one?

Reply:
We should use an example where files are obviously small, like 16mb

Reply:
128 was chosen because people copy the examples and 128MB is ideal size for most parquet based tables.

Reply:
Great .. so we should explain that this command is useful when a user observes many small files in the 16mb range or smaller .. and it gets it to the ideal size of 128MB .. ping me to work on wording together if you like @sheajamba